sql/update_util.sql

```diff
@@ -0,0 +1,40 @@
+-- This file and its contents are licensed under the Apache License 2.0.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-APACHE for a copy of the license.
+
+-- This file contains utility functions to handle post-update work.
+
+-- Procedure to update metadata for chunks compressed before 2.0
+CREATE OR REPLACE PROCEDURE update_compressed_metadata()
+LANGUAGE PLPGSQL AS
+$BODY$
+DECLARE
+  plain_chunk RECORD;
+  comp_chunk TEXT;
+  rowcount_pre BIGINT;
+  rowcount_post BIGINT;
+BEGIN
+  FOR plain_chunk IN
+  SELECT
+    *
+  FROM
+    _timescaledb_catalog.chunk comp
+  WHERE
+    compressed_chunk_id IS NOT NULL LOOP
+      SELECT
+        format('%I.%I', schema_name, table_name) INTO comp_chunk
+      FROM
+        _timescaledb_catalog.chunk
+      WHERE
+        id = plain_chunk.compressed_chunk_id;
+      EXECUTE format('SELECT sum(_ts_meta_count), count(*) FROM %s', comp_chunk) INTO rowcount_pre, rowcount_post;
```

Comment:
I'd recommend that we do a commit after each chunk here, there's no reason for it to be transactional and no reason to have one big long-running txn.

Reply:
Good point. However, it is only going to be used once after the update, so we will remove this function from the PR and just describe how to do it in the documentation.

Reply:
Hmmm...I think I'd keep the proc and have people run the proc in the documentation seems a lot simpler for the user. You can also check to see whether the chunk has nulls in the metadata table in the where clause here I think, that way it'll only run once.

Reply:
FWIW, we decided to move the script snippet to docs and not include in the extension. In docs, people can also adapt it for their own purposes (and we can easily modify it over time if needed).